feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt

```diff
@@ -109,8 +109,8 @@ private fun AboutScreen(
                     } else {
                         Text(
                             text = AboutStrings.Title.asString(),
-                            color = Color.Unspecified.copy(alpha = scrollBehavior.state.overlappedFraction),
```

Comment:
Text will decide with following code.
https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Text.kt;l=112-116?hl=ja
so, this copy of `Color.Unspecified` is not used.
Of course, we can specify color as `LocalContentColor.current.copy(alpha = scrollBehavior.state.overlappedFraction)` (refer to Text.kt)
but I used `Modifier.alpha(...)` because of code simplicity.
(if set Modifier, we don't have to care about COLOR of text, I considered)

Please comment if specifying color is better. (in that case, I'll change code)

Reply:
That makes sense!